.github/workflows/test-cleanup-workspace.yml

```diff
@@ -2,12 +2,10 @@ on:
   pull_request:
     paths:
       - ".github/workflows/test-cleanup-workspace.yml"
-      - ".github/actions/cleanup-workspace/**"
```

Comment:
Fun fact: this is not needed, because we use hash-pinning here, so changing the code won't have any effect, until you change the hash in this file (which I will later update), which will trigger this workflow